cmake/Modules/FindAubio.cmake

```diff
@@ -45,8 +45,10 @@ ExternalProject_Add(build-aubio-from-sources
         UPDATE_COMMAND ""
 	TMP_DIR ${CMAKE_BINARY_DIR}/3rdparty/aubio-build/tmp/
 	CONFIGURE_COMMAND ""
-	BUILD_COMMAND ${CMAKE_COMMAND} -E env CFLAGS=${AUBIO_CFLAGS} ${WAF}
-                  ${ACCELERATE_COMMAND} --disable-tests --notests --verbose
+	BUILD_COMMAND ${CMAKE_COMMAND} -E env CFLAGS=${AUBIO_CFLAGS}
+                  CXX=${CMAKE_CXX_COMPILER}
+                  CC=${CMAKE_C_COMPILER}
```

Comment:
Are CC and CXX needed? Normally they are already set to your local compiler setup 🤔

Reply:
that is the point, env is not inherited in Add_extranalProject so need to pass it (same as CXXFLAGS, actually)

Reply:
Ahhh fair enough! Kinda makes sense yea